### ISSUE TITLE
Media paths are not updating to new number prefix

### ISSUE THREAD

**Issue by joville** [getgrav/grav-plugin-admin#1500]
I have a modular page which haven't been number prefixed. After setting all modular pages to number prefixes and changing the order of the modular pages the media paths are not updating to the new prefix number. I am not sure whether is a grav core or grav admin bug. I have put this issue in the admin repo because of have changed the settings in the admin (grav 1.5.0 and grav-admin 1.8.8).

**Comment by joville** [comment]
Second issue: After setting the modular pages to use number prefixes, there are also remaining items in  the modular page:

        dir: asc
        custom:
            - _teaserimage
            - _features
            ...

After deleting this manually the ordering is getting right!

**Comment by rhukster** [comment]
your `custom` order is probably conflicting with the page order?  The custom order was really the 'old' way of doing things, numeric prefix is now the preferred way.

**Comment by joville** [comment]
Yes, I know! Was just, that saving with numbered prefix did not remove the old frontmatter setting (had to do it manually). Want to give you the information that this is happening.

**Comment by synyster0fa7x** [comment]
Hello there,

Same problem here, with build-in reorder admin functionnality.

![image](https://user-images.githubusercontent.com/1673917/45908379-a6a46500-bdfc-11e8-993e-ab00f90ad7ce.png)
I have this blueprints on my pages. When I reorder theses pages, media path are not updated (old prefix is still in my picture path)

**Comment by ishapkin** [comment]
> Hello there,
> 
> Same problem here, with build-in reorder admin functionnality.
> 
> ![image](https://user-images.githubusercontent.com/1673917/45908379-a6a46500-bdfc-11e8-993e-ab00f90ad7ce.png)
> I have this blueprints on my pages. When I reorder theses pages, media path are not updated (old prefix is still in my picture path)

Have same problem

**Comment by rhukster** [comment]
Have you guys cleared the cache????

**Comment by ishapkin** [comment]
Hi, thank you for answer!
Of course, i am cleared the cache before.
What i doing:

- Created some pages by template with some custom fields in blueprints:

`extends@: default
child_type: item

rules:
  slug:
    pattern: "[a-z][a-z0-9_\-]+"
    min: 2
    max: 80

form:
  fields:
    tabs:
      type: tabs
      active: 1

      fields:
        content:
          type: tab
          fields:

            header.media_order:
              unset@: true

            header.custom.avatar:
              type: file
              label: Cover image
              random_name: true
              destination: '@self'
              multiple: false
              accept:`

- After change page position with drag & drop
- The image what i loaded in this filed (_header.custom.avatar_) was lost in page.md
![image](https://user-images.githubusercontent.com/10353034/47739008-78306a00-dc85-11e8-9b1e-43e16079bc5a.png)
- In page.md i found old path for this file: 
![image](https://user-images.githubusercontent.com/10353034/47739244-0573be80-dc86-11e8-9325-d1ba8d95e6c0.png)

Now i changed `destination: '@self'` to  `destination: 'static'` and loading all files in static folder in webroot, but it was wrong way i think :)
